QStore4S/src/main/java/edu/asu/qstore4s/service/impl/Statistics.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.qstore4s.service.impl;
+
+import java.util.HashMap;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.neo4j.repository.GraphRepository;
+import org.springframework.stereotype.Service;
+
+import edu.asu.qstore4s.domain.events.impl.AppellationEvent;
+import edu.asu.qstore4s.domain.events.impl.CreationEvent;
+import edu.asu.qstore4s.exception.InvalidDataException;
+import edu.asu.qstore4s.repository.AppellationEventRepository;
+import edu.asu.qstore4s.repository.ElementNeo4jConnector;
+import edu.asu.qstore4s.repository.IElement;
+import edu.asu.qstore4s.repository.RelationEventRepository;
+import edu.asu.qstore4s.service.IStatistics;
+
+/**
+ * 
+ * @author Nikhil Aourpally A Service to get the statistics from QStore
+ */
+@Service
+public class Statistics implements IStatistics {
+
+    @Autowired
+    private AppellationEventRepository app;
```

Comment:
the variable should have a more descriptive name (same for the RelationEventRepository variable)